superset-frontend/src/components/TimezoneSelector/TimezoneSelector.test.tsx

```diff
@@ -54,9 +54,7 @@ describe('TimezoneSelector', () => {
     });
     expect(select).toBeInTheDocument();
     userEvent.click(select);
-    const selection = await screen.findByTitle(
-      'GMT -06:00 (Mountain Daylight Time)',
-    );
+    const selection = await screen.findByTitle('GMT -06:00 (America/Belize)');
```

Comment:
we should coverage "offsetsToName" in `src/components/TimezoneSelector/index.tsx`. I have a seprate [PR](https://github.com/apache/superset/pull/17370) do this.